shared/fs/common/open-in-files-tab-hoc.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+import * as React from 'react'
+import * as Types from '../../constants/types/fs'
+import * as FsGen from '../../actions/fs-gen'
+import {namedConnect} from '../../util/container'
+
+const mapDispatchToProps = dispatch => ({
+  onOpenInFilesTab: (path: Types.Path) => dispatch(FsGen.createOpenPathInFilesTab({path})),
+})
+
+export default (ComposedComponent: React.ComponentType<any>) =>
```

Comment:
Can you parametrize the `ComponentType<T>`? Components this is used on are losing their type.

```
    <BigTeamHeader
     ^ ComponentType<any>
```

Reply:
Then I think you shouldn't need to explicitly add `onOpenInFilesTab` to the props for the components